client/state/sites/actions.js

```diff
@@ -33,14 +33,12 @@ import { SITE_REQUEST_FIELDS, SITE_REQUEST_OPTIONS } from 'state/sites/constants
  * deleted.
  *
  * @param  {Number} siteId  ID of deleted site
- * @param  {Boolean} silent Indicates to not show the global notice after the site is removed from the state.
  * @return {Object}         Action object
  */
-export function receiveDeletedSite( siteId, silent = false ) {
```

Comment:
Oscar knew that we'll be removing this one eventually - that's why he didn't add a unit test for it 🔮 :trollface:

Reply:
I intended that one to never be refactored until the end of the humanity, therefore no unit tests !

Reply:
Nice motivation tho 🤣